src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/HttpHandlerDiagnosticListener.cs

```diff
@@ -0,0 +1,710 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections;
+using System.Collections.Generic;
+using System.Net;
+using System.Reflection;
+using System.Reflection.Emit;
+using System.Runtime.Serialization;
+
+// This HttpHandlerDiagnosticListener class is applicable only for .NET 4.6, and not for .NET core.
+// If you are making these changes, please test your changes manually via custom test applications.
```

Comment:
This comment is still confusing. What are "these changes"? Where are the "custom test applications"?